exonum-java-binding-fakes/src/main/java/com/exonum/binding/fakes/mocks/UserServiceAdapterMockBuilder.java

```diff
@@ -36,7 +36,29 @@
 @SuppressWarnings({"unused", "WeakerAccess"}) // Used in native code
 public final class UserServiceAdapterMockBuilder {
 
-  private final UserServiceAdapter service = mock(UserServiceAdapter.class);
+  private final UserServiceAdapter service;
+
+  /**
+   * Default constructor.
+   */
+  public UserServiceAdapterMockBuilder() {
+    this.service = mock(UserServiceAdapter.class);
+
+    // Some default mocking that could be overridden later
+    init_mocks();
+  }
+
+  private void init_mocks() {
+    byte[] hash = new byte[32];
+    for (int i = 0; i < 32; i++) {
+      hash[i] = (byte) i;
+    }
+
+    when(service.getId()).thenReturn((short) 42);
```

Comment:
Please reuse the methods that are already there (id, name, stateHashes) for clarity.